src/ImmutableX.ts

```diff
@@ -1,11 +1,14 @@
 import {
+  EthSigner,
   ImmutableXConfiguration,
-  WalletConnection,
-  UnsignedMintRequest,
+  NftTransferDetails,
   PrepareWithdrawalRequest,
   TokenDeposit,
   TokenWithdrawal,
-  EthSigner,
+  UnsignedMintRequest,
+  UnsignedOrderRequest,
+  UnsignedTransferRequest,
```

Comment:
Do these request types need "Unsigned" work in them? For the user of the SDK these are just requests and SDK internally handles the signing. Suggest removing the "Unsigned"

Reply:
Would clash with generated api model names.